src/core/ext/transport/chttp2/client/secure/secure_channel_create.c

```diff
@@ -97,15 +98,15 @@ grpc_channel *grpc_secure_channel_create(grpc_channel_credentials *creds,
   GRPC_API_TRACE(
       "grpc_secure_channel_create(creds=%p, target=%s, args=%p, "
       "reserved=%p)",
-      4, (creds, target, args, reserved));
+      4, ((void *)creds, target, (void *)args, (void *)reserved));
   GPR_ASSERT(reserved == NULL);
   // Make sure security connector does not already exist in args.
```

Comment:
Went with option 1. A nice side effect of this is that grpclb needn't be split in secure and insecure versions now.